test/powershell/engine/Basic/SemanticVersion.Tests.ps1

```diff
@@ -188,7 +196,6 @@ Describe "SemanticVersion api tests" -Tags 'CI' {
             @{ name = "Missing parts";               errorId = "FormatException";    expectedResult = $false; version = "1.0.+alpha" }
             @{ name = "Missing parts";               errorId = "FormatException";    expectedResult = $false; version = "1.0.0-alpha+" }
             @{ name = "Missing parts";               errorId = "FormatException";    expectedResult = $false; version = "1.0.0-+" }
-            @{ name = "Missing parts";               errorId = "FormatException";    expectedResult = $false; version = "1.0.0+-" }
             @{ name = "Missing parts";               errorId = "FormatException";    expectedResult = $false; version = "1.0.0+" }
             @{ name = "Missing parts";               errorId = "FormatException";    expectedResult = $false; version = "1.0.0-" }
             @{ name = "Missing parts";               errorId = "FormatException";    expectedResult = $false; version = "1.0.0." }
```

Comment:
Please add the following negative cases missing in build metadata and prerelease label

1. non-ascii characters: 🤷‍♂️
2. other symbols, for example: &